tests/vmi_networking_test.go

```diff
@@ -189,29 +189,19 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				virtClient,
 				vmiPod,
 				"compute",
-				[]string{"ip", "address", "show", "k6t-eth0"},
+				[]string{"cat", "/sys/class/net/k6t-eth0/mtu"},
```

Comment:
Why do we need this change? `ip address show` is not working anymore?

Reply:
it does, but we just want the mtu, so no need to parse the whole line
it was Suggested by Dan